puphpet/config.yaml.template

```diff
@@ -79,25 +100,31 @@ apache:
     settings:
         user: www-data
         group: www-data
-        default_vhost: true
+        default_vhost: false
```

Comment:
It isn't always bad, but traditionally we've used the root level `.htaccess` file to specify apache configs for production that we don't have to worry about in vagrant. Stuff like `php_value memory_limit` where we don't have access to apache's raw config files.

Now granted, we shouldn't be hosting 3.0 projects in those kinds of environments anymore, but the separate between pointing production to `/var/www` (essentially) and vagrant to `/var/www/webroot` means that we can put production configs in `ROOT/.htaccess` and not get conflicts in vagrant. It really shouldn't come up much, but it's handy when it does and it's nice to have it set up that way already. 

For "proper" hosting, we would instead specify the necessary configs directly in the <VirtualHost> config but this set up doesn't hurt or impact that in any way.